test/powershell/Modules/Microsoft.PowerShell.Utility/Implicit.Remoting.Tests.ps1

```diff
@@ -2104,17 +2125,15 @@ Describe "Import-PSSession on Restricted Session" -tags "Feature","RequireAdminO
         else
         {
             if ($session -ne $null) { Remove-PSSession -Session $session -ErrorAction SilentlyContinue }
-            Unregister-PSSessionConfiguration -Name restricted -Force -ErrorAction SilentlyContinue
+            Unregister-PSSessionConfiguration -Name $configName -Force -ErrorAction SilentlyContinue
         }
     }
 
-    # Blocked by https://github.com/PowerShell/PowerShell/issues/4275
-    # Need a way to created restricted endpoint based on a different endpoint other than microsoft.powershell which points
-    # to Windows PowerShell 5.1
-    It "Verifies that Import-PSSession works on a restricted session" -Pending {
+    It "Verifies that Import-PSSession works on a restricted session" {
 
         $errorVariable = $null
-        Import-PSSession -Session $session -AllowClobber -ErrorVariable $errorVariable
+        $module = Import-PSSession -Session $session -AllowClobber -ErrorVariable $errorVariable -CommandName Get-Help
         $errorVariable | Should BeNullOrEmpty
+        if ($module -ne $null) { Remove-Module $module -Force -ErrorAction SilentlyContinue }
```

Comment:
Since the `Remove-Module` is part of cleanup and not the test, convention is to have the test in a try{} and the cleanup in a finally{}.  No need to check if "$module -ne $null", in general, we just call `Remove-Module -Force -ErrorAction SilentlyContinue`

Reply:
I feel the null check is needed since the Cmdlet will throw an exception on a null parameter.  Otherwise fixed.